config.toml

```diff
@@ -0,0 +1,17 @@
+title = "Rust Game Development Working Group"
+description = "Stay up to date with the progress and recent developments in the Rust Game Development Working Group."
+base_url = "https://rust-gamedev.github.io/"
+default_language = "en"
+compile_sass = true
+highlight_code = true
+generate_rss = true
+
+[extra]
+date_format = "%b %-d, %Y"
```

Comment:
Can we get a more international YYYY-MM-DD?

Reply:
Could do, although I think it kinda looks a bit uglier:

![image](https://user-images.githubusercontent.com/784533/69761793-504d5780-1160-11ea-9ff4-1c5192fa6f23.png)

Will wait and see what other people think, as this would be a change from how things look on the live site.

(All dates are YYYY-MM-DD in metadata, etc, this one is just for the display formatting)

Reply:
@ozkriff thumbs up'ed your comment, so :+1: